pkg/cmd/time-entry/report/util/reportwithrange_test.go

```diff
@@ -246,6 +246,7 @@ func TestReportWithRange(t *testing.T) {
 			},
 			flags: func(t *testing.T) util.ReportFlags {
 				rf := util.NewReportFlags()
+				rf.TimeZone = "UTC"
```

Comment:
add another entry in the table, but with the following format:
```go
rf.Format = "{{.ID}};{{ .TimeInterval.Start.Format " +
	`"2006-01-02 15:04:05"` +
	" }}"
```

and a offset timezone so we can see the effect here.

Reply:
@lucassabreu thanks for the comments, I have some strange bugs with this feature (the date-filling flag), I will try to fix them this week.